packages/ember-runtime/lib/system/core_object.js

```diff
@@ -191,6 +191,10 @@ function makeCtor() {
       this.init.apply(this, args);
     }
 
+    if (this.__postInitInitialization) {
```

Comment:
I'm slightly concerned that the property miss will not be as ideal as we would want, although I suspect today it is just in the noise.